ci/code_checks.sh

```diff
@@ -120,9 +120,15 @@ if [[ -z "$CHECK" || "$CHECK" == "patterns" ]]; then
     # Check for imports from pandas.core.common instead of `import pandas.core.common as com`
     # Check for imports from collections.abc instead of `from collections import abc`
     MSG='Check for non-standard imports' ; echo $MSG
-    invgrep -R --include="*.py*" -E "from pandas.core.common import " pandas
-    invgrep -R --include="*.py*" -E "from collections.abc import " pandas
-    invgrep -R --include="*.py*" -E "from numpy import nan " pandas
+    invgrep -R --include="*.py*" -E "from pandas.core.common import" pandas
+    invgrep -R --include="*.py*" -E "from pandas.core import common" pandas
```

Comment:
This pattern is used in the code base, so I fixed the existing cases (not a breaking change) and added this linting rule here. Let me know if that's fine with you